pkg/audit/annotation.go

```diff
@@ -0,0 +1,41 @@
+package audit
+
+import (
+	"net/http"
+
+	kaudit "k8s.io/apiserver/pkg/audit"
+)
+
+// Decision is a particular type to prevent accidential mis-match of string values.
+type Decision string
+
+const (
+	UsernameAnnotation = "authentication.openshift.io/username"
+	DecisionAnnotation = "authentication.openshift.io/decision"
+
+	// AllowDecision is logged on a successful authentication.
+	AllowDecision Decision = "allow"
+	// DenyDecision is logged on an unsuccessful authentication attempt.
+	DenyDecision Decision = "deny"
+	// ErrorDecision is logged on errors that might not relate to the authentication itself.
+	ErrorDecision Decision = "error"
+)
+
+// String returns the string representation of the Decision.
+func (d Decision) String() string {
```

Comment:
since this is an alias why is this needed?

Reply:
Done